### PR TITLE
core(font-size): recalibrate the legible font sizes

### LINKED ISSUE #4448: Recalibrate the legible font sizes audit
Apparently 85% of my site is not lisible because the font-size is smaller than 16px. Or the font size is almost everywhere 18px...and it is lisible on mobile

### PR DESCRIPTION
**Pull request by kdzwinel**
Fixes #4448

> Minimum legible font size: 12px (was 16px)
> Minimum percent legibility: 60% (was 75%)

### CODE REVIEW

File: lighthouse-cli/test/fixtures/seo/seo-tester.html [hunk 1]
```diff
@@ -36,9 +36,9 @@ <h2>Anchor text</h2>
   <a href='javascript:void(0);'>click this</a>
 
   <h2>Small text</h2>
-  <!-- PASS(font-size): amount of illegible text is below the 75% threshold -->
+  <!-- PASS(font-size): amount of illegible text is below the 60% threshold -->
   <p class='small'> 1 </p>
-  <small>2</small>
```
Comment (kdzwinel) [inline lighthouse-cli/test/fixtures/seo/seo-tester.html]: `<small>` has a font-size of ~13px , h6 is ~10px

File: lighthouse-core/audits/seo/font-size.js [hunk 2]
```diff
@@ -177,9 +177,9 @@ class FontSize extends Audit {
       name: 'font-size',
       description: 'Document uses legible font sizes',
       failureDescription: 'Document doesn\'t use legible font sizes',
-      helpText: 'Font sizes less than 16px are too small to be legible and require mobile ' +
-      'visitors to “pinch to zoom” in order to read. Strive to have >75% of page text ≥16px. ' +
-      '[Learn more](https://developers.google.com/speed/docs/insights/UseLegibleFontSizes).',
+      helpText: 'Font sizes less than 12px are too small to be legible and require mobile ' +
+      'visitors to “pinch to zoom” in order to read. Strive to have >60% of page text ≥12px. ' +
+      '[Learn more](https://developers.google.com/web/fundamentals/design-and-ux/responsive/#optimize_text_for_reading).',
```
Comment (kdzwinel) [inline lighthouse-core/audits/seo/font-size.js]: @rviscomi FYI I updated the link to avoid the redirect + BTW do we have anything better we can link to?

Comment (rviscomi) [reply lighthouse-core/audits/seo/font-size.js]: This is ok for now. Please work with @kaycebasques to get first-party LH SEO audit docs online and we can update this link when ready.

**Comment by pingram3541** [comment]
Curious, do sites actually get penalized for this via regular scan algorithms using this rule set (actual Google ranking) or is this only informative through the Lighthouse audit other tools like Search Console notifications?

**Pull request merged**